build_tools/cmake/iree_external_cmake_options.cmake

```diff
@@ -4,6 +4,46 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+include(CMakeDependentOption)
+
+#-------------------------------------------------------------------------------
```

Comment:
can we put this in a dedicated file (compiler_options.cmake or something) instead? I'd not expect the core options for our project (whether to build the compiler, or python bindings, etc) to be buried in iree_external_cmake_options.cmake.

Reply:
(a compiler_options.cmake and runtime_options.cmake would be nice - easy to find/refer people to, just our stuff so no cross contamination)

Reply:
Ok, was debating that.